catchup/catchpointService.go

```diff
@@ -156,11 +160,16 @@ func MakeNewCatchpointCatchupService(catchpoint string, node CatchpointCatchupNo
 }
 
 // Start starts the catchpoint catchup service ( continue in the process )
-func (cs *CatchpointCatchupService) Start(ctx context.Context) {
+func (cs *CatchpointCatchupService) Start(ctx context.Context) error {
```

Comment:
@algorandskiy this function returning an error is the primary functional change. Today, if this function fails the catchup fails silently. By returning an error the API can return the error and show it to the user.